.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "mcux/mcux-sdk"]
+	path = mcux/mcux-sdk
+	url = https://github.com/NXPmicro/mcux-sdk.git
```

Comment:
Shouldn't this pull from a zephyrproject-rtos/mcux-sdk.git repo instead? How do we set which revision of the mcux-sdk repo should be pulled in?

Reply:
Right now we have it set to track the tip of `mcux-sdk` github master branch as we need Zephyr specific changes that got added post MCUX-SDK 2.10 release. Going forward we would update the `commit` to the latest SDK release pushed to `mcux-sdk` github. However we will have to discuss what happens when a patch is released on the `mcux-sdk` github post SDK releases.

Reply:
Great process questions which we will work on. There is a readme in the mcux-sdk that we can update with the process to be more transparent. But ideally, we will update the Zephyr HAL pointers when there is a major release to the mcux-sdk and when there are patches to the mcux-sdk (from internal NXP or external community folks).

Reply:
@henrikbrixandersen I have updated this PR to pull in `mcux-sdk` repo as a `git subtree` and not a `submodule`. This would make a copy of the `mcux-sdk` repo in Zephyr HAL